test/errors/test_FPDF_errors.py

```diff
@@ -21,9 +21,11 @@ def test_encoding_exception():
     pdf = fpdf.FPDF()
     pdf.add_page()
     pdf.set_font("Helvetica", size=15)
-    with pytest.raises(FPDFUnicodeEncodingException):
+    with pytest.raises(FPDFUnicodeEncodingException) as e:
         pdf.cell(txt="Joséō")
         # This should through an error since Helvetica is a latin-1 encoder and the ō is out of range.
+    msg = 'Character "ō" at index 4 in text is outside the range of characters supported by the font used: "helvetica". Please consider using a font that supports UTF-8 encoding.'
```

Comment:
While you're at it right here, could you maybe also correct that error message?
It should simply say "Please consider using a Unicode font". Unlike traditional fonts, Unicode fonts are not specific to any encoding (such as "utf-8").

Reply:
Should I change it on errors.py where the exception is defined or just change the test case?
https://github.com/PyFPDF/fpdf2/blob/3c29cbab17fdfec17efcc5c39ab99440291c6556/fpdf/errors.py#L47-L52

Reply:
Yes, the change must be made in `errors.py` :)